akka-stream-testkit/src/test/scala/akka/stream/testkit/StreamTestKitSpec.scala

```diff
@@ -121,13 +122,17 @@ class StreamTestKitSpec extends AkkaSpec {
     "#expectNextWithTimeoutPF should fail after timeout when element delayed" in {
       intercept[AssertionError] {
         val timeout = 100.millis
-        val overTimeout = timeout + 50.millis
+        // Initial delay is over the timeout so an exception will be thrown.
+        // It also needs to be dilated since the testkit will eventually dilate
+        // the timeout accordingly to `-Dakka.test.timefactor` value.
```

Comment:
not essential for the PR, but I found the text here confusing. For example, "Initial delay is over the timeout". Do you mean "Initial delay is higher than the timeout"?

Also, why you say it will eventually dilate the timeout. I think you mean that it will dilate the timeout, no?

Reply:
@octonato, see the comments now. I think they are clearer now.